src/components/LoginModal/LoginModal.js

```diff
@@ -117,10 +101,6 @@ class LoginModal extends Component {
               rules: [{ required: true, message: 'Please input your email!' }],
             })(
               <Input
-                type="email"
```

Comment:
@ritwizsinha  Add this line `type="email"` . Rest is good 👍

Reply:
doing